experimental/handoff-library/csharp_dotnetcore/samples/LivePersonAgentBot/PostDeployScripts/runGulp.cmd

```diff
@@ -0,0 +1,23 @@
+@echo off
```

Comment:
I think I'm not understanding what these CMD files are for. Again ,they all won't run on Mac/Linux which is something we need our samples to do.